changelog.d/pr-7402

```diff
@@ -6,8 +6,7 @@ issues: #5591 #7180 #7405
 description: {
 
 - Removes the --dry-run part of the v2-configure command
-- Adds an --append flag to v2-configure, which allows new configuration to be appended to the old config file
-- Adds a --backup flag to v2-configure, which enables the backup feature for the config file
-- Includes unit tests for everything
```

Comment:
I removed this line because it doesn't have much context and it's also not a user-facing change.